src/RateLimit.ts

```diff
@@ -116,7 +116,7 @@ export class RateLimit {
      * Clear rate limit attempts storage. This is equivalent to resetting all rate limits.
      */
     clear(): void {
-        if (this.#deleted) throw new Error(`Rate limit "${this.#name}" has been deleted. Construct a new instance`);
+        if (this.#deleted) throw new Error(`Rate limit "${this.name}" has been deleted. Construct a new instance`);
```

Comment:
are you sure its this.#deleted and not "this.deleted" just asking

Reply:
yes, it's a private field. private fields begin with `#`

Reply:
alr cheers